R/cr_cn.r

```diff
@@ -104,6 +110,28 @@
 #'    style = "plant-cell-and-environment")
 #' cr_cn("10.5284/1011335", url = "https://data.datacite.org", 
 #'    style = "plant-cell-and-environment")
+#' 
+#' # with caching
+#' ## never expire
+#' webmiddens::wm_enable()
+#' webmiddens::wm_configuration("foostuff")
+#' cr_cn(dois="10.1126/science.169.3946.635", verbose=TRUE)
+#' cr_cn(dois="10.1002/app.27716", verbose=TRUE)
+#' 
+#' (dois <- cr_r(10))
+#' system.time((b=cr_cn(dois)))
```

Comment:
you could add a comment indicating one could also turn off wifi and it'd still work?

Reply:
will do